toolkit/scss/shared_placeholders/_temporary-messages.scss

```diff
@@ -6,4 +6,13 @@
   background: $govuk-blue;
   color: $white;
   padding: $gutter-half $gutter-half 18px $gutter-half;
+
+  a, a:visited {
+    color: $white;
+  }
+
+  a:hover, a:active {
+    @extend a;
+    text-decoration: none;
```

Comment:
Does this remove the underline when the link is hovered over?  I don't think that happens with any of our other links - they change colour but remain underlined.  It seems odd to change that pattern just for these links?